docs/dev_guide/index.rst

```diff
@@ -12,11 +12,11 @@ If you would rather install the latest development version, and/or compile direc
 
 .. code:: sh
 
-   $ git clone https://github.com/quantumjot/BayesianTracker.git
-   $ conda env create -f ./BayesianTracker/environment.yml
-   $ conda activate btrack
-   $ cd BayesianTracker
-   $ pip install -e .
+   git clone https://github.com/quantumjot/BayesianTracker.git
+   conda env create -f ./BayesianTracker/environment.yml
+   conda activate btrack
+   cd BayesianTracker
+   pip install -e .
```

Comment:
I'm not sure, but do we want to mention that it's `pip install -e .[docs]` if developing documentation anywhere?

Reply:
I think it's worth adding this. Might add a new issue for this.